examples/01-getting-started.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>The function simplifies the demonstration, but isn't a key feature of the library. Maybe...</p><p> </p><p>We run the the <code>get_movielens</code> function as a convenience to download the dataset, perform simple preprocessing, and split the data into training and validation datasets.</p><p> </p><p>(I'll never suggest "We" in any context other than a notebook, and that's only because it's the prevailing etiquette.)  If there's no other preprocessing than creating the datasets, remove the suggestion of preprocessing.  If there's some operation being performed, can you add a hint about what is done and why it might be useful, helpful, or necessary for the DLRM model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I updated the the text.</p><p> </p><p>I did not add additional information about the preprocessing. I think, this is a getting started notebook and should be pretty lean. The focus is to see how easy it is to use Merlin Models for training (if the dataset is prepared). The following notebooks explain/explore more what is necessary</p>